backends/gstreamer/render-unix/lib.rs

```diff
@@ -0,0 +1,193 @@
+#![cfg(any(
+    target_os = "linux",
+    target_os = "dragonfly",
+    target_os = "freebsd",
+    target_os = "netbsd",
+    target_os = "openbsd"
+))]
+
+extern crate glib;
+#[macro_use]
+extern crate gstreamer as gst;
+use gst::prelude::*;
+extern crate gstreamer_gl as gst_gl;
+use gst_gl::prelude::*;
+extern crate gstreamer_video as gst_video;
+
+extern crate servo_media_gstreamer_render as sm_gst_render;
+use sm_gst_render::Render;
+
+extern crate servo_media_player as sm_player;
+use sm_player::frame::{Buffer, Frame, FrameData};
+use sm_player::{GlContext, PlayerError};
```

Comment:
uber-nit: could we group the `extern`s and the `use`s in separate blocks, please?